index.html

```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        
+        <!-- ________ *Rafael* Contributor card START ________  -->
+      <div class="card">
+        <p class="name">Rafael José Jiménez</p>
+        <p class="contact">
+          <i class="fab fa-twitter"></i>
+          <a href="https://twitter.com/fistan85" target="_blank">@fistan85</a>
+        </p>
+        <p class="about">...</p>
+        <div class="resources">
+          <p>3 Useful Dev Resources</p>
+          <ul>
+            <li>
+              <a href="#" target="_blank" title="First Resource">Resource 1</a>
```

Comment:
Kindly added the resources
`<a href=`"Your resource URL", `title=`"Resorce Title and Put the resource name in the place of `Resource 1`

add 3 resources to complete your card